### PR TITLE
Diversity and Inclusion Sub-Team

### PR DESCRIPTION
**Pull request by CJ-Wright**
This language is very much open to edits, critique and discussion, everyone is encouraged to participate!

If you have write access (most likely if you are on core) and would like changes (including adding yourself) please feel free to push. Otherwise please feel free to leave comments and such.

### CODE REVIEW

**Comment by pkgw** [comment]
I'd like to participate in this team, if that's compatible with the fact that I have a standing meeting that prevents me from attending the conda-forge core telecons. Would you mind adding me to the roster?

**Comment by CJ-Wright** [comment]
@pkgw Can you push to my PR? I think this team will have independent meetings from the general core meeting.

**Comment by marcelotrevisani** [comment]
I would like to participate as well

File: src/orga/subteams.rst [hunk 1]
```diff
@@ -264,3 +264,34 @@ Members
...
+community at all levels.
+As such this group can, but is not limited to:
+- provide a place for issues impacting diversity to be heard
+- advocate for underrepresented groups and bring their issues to the attention of core
+- run programs to maintain and grow the community's diversity and inclusiveness
+
+Members
+-------
+- Filipe Fernandes <ocefpaf@gmail.com>
+- Christopher J. "CJ" Wright <cjwright4242@gmail.com>
```
Comment (marcelotrevisani) [inline src/orga/subteams.rst:296]: - Marcelo Duarte Trevisani <marceloduartetrevisani@gmail.com>

**Comment by marcelotrevisani** [comment]
Thanks! :)

File: src/orga/subteams.rst [hunk 2]
```diff
@@ -264,3 +264,35 @@ Members
...
+- Marcelo Duarte Trevisani <marceloduartetrevisani@gmail.com>
+- Peter K. G. Williams <peter@newton.cx>
```
Comment (kkraus14) [inline src/orga/subteams.rst:298]:
```suggestion
- Peter K. G. Williams <peter@newton.cx>
- Keith Kraus <kkraus@nvidia.com>
```

File: src/orga/subteams.rst [hunk 3]
```diff
@@ -264,3 +264,35 @@ Members
...
+actions impacting underrepresented groups in ``conda-forge``.
+
+Charter
+-------
+Dynamic
+
+Responsibility
+--------------
+One of the core strengths of ``conda-forge`` is the diversity of ecosystems it supports.
+Likewise, fostering and advancing a diverse community of users, maintainers and infrastructure contributors
```
Comment (kkraus14) [inline src/orga/subteams.rst:284]:
```suggestion
Likewise, fostering and advancing a diverse community of users, maintainers, and infrastructure contributors
```

**Review by kkraus14 — APPROVED** [review]
lgtm other than an oxford comma 😄

I'd also like to participate

**Review by marcelotrevisani — APPROVED** [review]

**Review by jakirkham — APPROVED** [review]

**Review by beckermr — APPROVED** [review]

**Review by xhochy — APPROVED** [review]

**Review by ocefpaf — APPROVED** [review]

**Review by pkgw — APPROVED** [review]

**Comment by CJ-Wright** [comment]
I'm glad for the support (both in votes and in membership) from core, however I don't think this should be merged until we have some other perspectives on this team.

**Review by chrisburr — APPROVED** [review]

**Review by djsutherland — APPROVED** [review]

**Review by mwcraig — APPROVED** [review]

**Comment by CJ-Wright** [comment]
At this point I think it would be good to move forward with this proposal. It would be good to add more people to the team and I look forward to doing that once it is set up!


@conda-forge/core
This PR falls under the Sub-team Formation policy, please vote and/or comment on this PR.
This PR needs 50% of core to vote yea to pass.
To vote please leave Approve (yea) or Request Changes (nay) reviews.
If you would like changes to the current language please leave a comment or push to this branch.
This vote will end on April 14th.


I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.

**Review by dopplershift — APPROVED** [review]

**Review by kkraus14 — APPROVED** [review]

**Review by ocefpaf — APPROVED** [review]

**Review by viniciusdc — APPROVED** [review]

**Review by ericdill — APPROVED** [review]

**Comment by CJ-Wright** [comment]
This vote has passed!

**Pull request merged**